py7zr/archiveinfo.py

```diff
@@ -576,14 +602,15 @@ def write(self, file: BinaryIO, numfolders: int):
             write_byte(file, Property.NUM_UNPACK_STREAM)
             for n in self.num_unpackstreams_folders:
                 write_uint64(file, n)
-        write_byte(file, Property.SIZE)
-        idx = 0
-        for i in range(numfolders):
-            for j in range(1, self.num_unpackstreams_folders[i]):
-                size = self.unpacksizes[idx]
-                write_uint64(file, size)
+        if self.write_unpacksizes:
```

Comment:
p7zip does not produce unpacksizes field of substreamsinfo, so here is a flag for debug.

Reply:
revert this part.